odoo/addons/test_main_flows/static/src/js/tour.js

```diff
@@ -407,14 +407,16 @@ tour.register('main_flow_tour', {
 }, {
 // Run the schedulers
     edition: "enterprise",
-    trigger: ".o_menu_sections a:contains('Schedulers')",
-    content: _t("Go to Schedulers"),
+    trigger: ".o_menu_sections a:contains('Operations')",
+    content: _t("Go to Scheduler"),
     position: "bottom"
 }, {
-    trigger: ".o_menu_sections a:has(span:contains('Run Schedulers')), .oe_secondary_submenu .oe_menu_text:contains('Run Schedulers'):first",
-    content: _t("Run the schedulers"),
+    edition: "enterprise",
+    trigger: ".o_menu_sections a:has(span:contains('Run Scheduler')), .oe_secondary_submenu .oe_menu_text:contains('Run Scheduler'):first",
+    content: _t("Run the scheduler"),
```

Comment:
This should work without modification I think. The text on the button in the wizard still says 'Run Scheduler**s**'. Also why did you add `edition: "enterprise",`?